packages/govern-subgraph/src/Govern.ts

```diff
@@ -7,23 +7,17 @@ import {
 } from '../generated/templates/Govern/Govern'
 import {
   Govern as GovernEntity,
-  Execution as ExecutionEntity
 } from '../generated/schema'
 import { frozenRoles, roleGranted, roleRevoked } from './lib/MiniACL'
 
 export function handleExecuted(event: ExecutedEvent): void {
   const govern = loadOrCreateGovern(event.address)
 
-  const execution = loadOrCreateExecution(event)
+  const currentContainers = govern.containers
 
-  execution.results = event.params.execResults
```

Comment:
This is the only place we can keep track of the results of the execution. Maybe have it as part of the `Execute` container event.

Reply:
Yeah, definitely agree. :)